srcCs/PerftestTransport.cs

```diff
@@ -877,56 +903,118 @@ private void ConfigureSecurityFiles(DDS.DomainParticipantQos qos)
 
         public bool ConfigureTransport(DDS.DomainParticipantQos qos) {
 
+            /*
+             * If transportConfig.kind is not set, then we want to use the value
+             * provided by the Participant Qos, so first we read it from there and
+             * update the value of transportConfig.kind with whatever was already set.
+             */
+            if (transportConfig.kind == Transport.TRANSPORT_NOT_SET)
+            {
+                int mask = qos.transport_builtin.mask;
+                switch (mask)
+                {
+                    case (int)TransportBuiltinKind.TRANSPORTBUILTIN_UDPv4:
+                        transportConfig = new TransportConfig(
+                                Transport.TRANSPORT_UDPv4,
+                                "UDPv4",
+                                "dds.transport.UDPv4.builtin");
+                        break;
+                    case (int)TransportBuiltinKind.TRANSPORTBUILTIN_UDPv6:
+                        transportConfig = new TransportConfig(
+                            Transport.TRANSPORT_UDPv6,
```

Comment:
add 4 spaces here and in the following functions.

Reply:
Done.